tools/fetch_source_files.py

```diff
@@ -229,26 +223,33 @@ def copy(item: str, src: str, dst: str, rel_path: str):
                 dest_path = os.path.join(ROOT_DIR, "taipy")
                 if not os.path.exists(dest_path):
                     os.makedirs(dest_path)
-                src_path = os.path.join(src_path, "src", "taipy")
-                for item in os.listdir(src_path):
-                    copy(item, src_path, dest_path, "")
+                sources_path = os.path.join(src_path, "taipy")
```

Comment:
evaluate the opportunity of using pathlib.Path for clarity ?
- Path(ROOT_DIR) / "taipy"
- Path(...).exists()
- Path(...).iterdir() ...

Reply:
Will do. Just not now. Whatever works. Thx